agents/sorenson_dlm/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# SOCS DLM Agent
```

Comment:
For this docker image to automatically build once we merge this also needs to be added to the main `docker-compose.yaml` file. Add a block like:

```
  ocs-dlm-agent:
    image: "ocs-dlm-agent"
    build: ./agents/sorenson_dlm/
    depends_on:
      - "socs"
```

Reply:
Can you clarify where this should be added? I checked other agents' `Dockerfile` and do not see this block in there.

Reply:
Yup, sorry, over here: https://github.com/simonsobs/socs/blob/develop/docker-compose.yml